sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKey.cs

```diff
@@ -120,6 +120,250 @@ public JsonWebKey(string keyId, string keyType, IList<string> keyOperations)
+                X = Encoding.ASCII.GetBytes(x.GetString());
+            }
+            if (json.TryGetProperty("y", out JsonElement y))
+            {
+                Y = Encoding.ASCII.GetBytes(y.GetString());
+            }
+            if (json.TryGetProperty("d", out JsonElement d))
+            {
+                D = Encoding.ASCII.GetBytes(d.GetString());
+            }
+            if (json.TryGetProperty("k", out JsonElement k))
+            {
+                K = Encoding.ASCII.GetBytes(k.GetString());
+            }
+            if (json.TryGetProperty("t", out JsonElement t))
+            {
+                T = Encoding.ASCII.GetBytes(t.GetString());
+            }
+        }
+
+        internal override void WriteProperties(ref Utf8JsonWriter json)
+        {
+            if (KeyType != default)
+            {
+                json.WriteString("kty", KeyTypeExtensions.ParseToString(KeyType));
+            }
+            if (KeyOps != null)
+            {
+                json.WriteStartArray("key_ops");
+                foreach (var operation in KeyOps)
+                {
+                    json.WriteStringValue(KeyOperationsExtensions.ParseToString(operation));
+                }
+                json.WriteEndArray();
+            }
+            if (!string.IsNullOrEmpty(CurveName))
+            {
+                json.WriteString("crv", CurveName);
+            }
+            if (N != null)
+            {
+                json.WriteString("n", Encoding.ASCII.GetString(N));
+            }
+            if (E != null)
+            {
+                json.WriteString("e", Encoding.ASCII.GetString(E));
+            }
+            if (DP != null)
+            {
+                json.WriteString("dp", Encoding.ASCII.GetString(DP));
+            }
+            if (DQ != null)
+            {
+                json.WriteString("dq", Encoding.ASCII.GetString(DQ));
+            }
+            if (QI != null)
+            {
+                json.WriteString("qi", Encoding.ASCII.GetString(QI));
+            }
+            if (P != null)
+            {
+                json.WriteString("p", Encoding.ASCII.GetString(P));
+            }
+            if (Q != null)
+            {
+                json.WriteString("q", Encoding.ASCII.GetString(Q));
+            }
+            if (X != null)
+            {
+                json.WriteString("x", Encoding.ASCII.GetString(X));
+            }
+            if (Y != null)
+            {
+                json.WriteString("y", Encoding.ASCII.GetString(Y));
+            }
+            if (D != null)
+            {
+                json.WriteString("d", Encoding.ASCII.GetString(D));
+            }
+            if (K != null)
+            {
+                json.WriteString("k", Encoding.ASCII.GetString(K));
+            }
+            if (T != null)
+            {
+                json.WriteString("t", Encoding.ASCII.GetString(T));
+            }
+        }
+
+        public bool Equals(JsonWebKey other)
```

Comment:
Do we have a scenario for Equality comparison in this type?

Reply:
hmm good point. Not so far. I copied this from Track 1, it should be ok to delete.